components/data-bridge/org.wso2.carbon.databridge.core/src/main/java/org/wso2/carbon/databridge/core/DataBridge.java

```diff
@@ -437,6 +444,11 @@ public boolean deleteStreamDefinition(String streamName,
         return streamDefinitionStore.deleteStreamDefinition(streamName, streamVersion, tenantId);
     }
 
+    @Override
```

Comment:
Why do we need to have this in the databridge API? I don't think this is necessary.